README.md

```diff
@@ -0,0 +1,67 @@
+# Bambora
+
+The officiall Bambora Ruby library is not thread-safe. This means you will run into errors when using it with Sidekiq or Puma. This gem is a thread-safe client for the Bambora and Beanstream APIs.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'bambora'
```

Comment:
should it include `himama`?

Reply:
Good question. The `bambora` isn't claimed on ruby gems. Is it rude to take their name?

Reply:
i meant some source targeting this very repo should be mentioned :)

Reply:
@erawk Do you have thoughts on publishing this to rubygems.org? If we do that, should we name it `bambora`?

Reply:
Maybe `bambora-client` ?